wallet-wasm/src/lib.rs

```diff
@@ -1274,6 +1274,32 @@ pub extern "C" fn redemption_private_to_address(
     return address_bytes.len() as u32;
 }
 
+
+#[derive(Serialize, Deserialize, Debug)]
+struct RedemptionAvvmTxOutput {
+    tx_id: tx::TxId,
+    address: Vec<u8>,
+}
+
+#[no_mangle]
+pub extern "C" fn redemption_public_to_avvm_tx_out(
+    public_ptr: *const c_uchar,
+    protocol_magic: u32,
+    output_ptr: *mut c_uchar,
+) -> i32 {
+    let pub_key = unsafe {
+        let slice: &[u8] = std::slice::from_raw_parts(public_ptr, redeem::PUBLICKEY_SIZE);
+        redeem::PublicKey::from_slice(slice).unwrap()
+    };
+    let magic = cardano::config::ProtocolMagic::from(protocol_magic);
+    let (tx, address) = tx::redeem_pubkey_to_txid(&pub_key, magic);
+    let address_bytes = cbor!(address).unwrap();
```

Comment:
same here: use the `jrpc_try!` macro instead of the unwrap. If there is an error we need to find out quickly enough